SoftLayer/CLI/loadbal/detail.py

```diff
@@ -13,8 +14,15 @@
 def cli(env, identifier):
     """Get Load Balancer as a Service details."""
     mgr = SoftLayer.LoadBalancerManager(env.client)
-    _, lbid = mgr.get_lbaas_uuid_id(identifier)
-    this_lb = mgr.get_lb(lbid)
+
+    if utils.valid_domain(identifier):
+        lbaas = mgr.get_lbaas_by_address(identifier)
+        if not lbaas:
+            raise exceptions.CLIAbort("{} address not found".format(identifier))
+        this_lb = mgr.get_lb(lbaas.get('id'))
```

Comment:
Do we need to get the lb again after getting it the first time by address?

Reply:
get_lb is adding properties by using a mask, and member health by getLoadBalancerMemberHealth.

Reply:
The reason we have 1 api call to just get the Id/uuid is the LBaaS apis are somewhat inconsistent on when they require an id, and when they require a UUID. So I felt this approach made the lbaas manager simpler and more internally consistent, although I'll likely think of a better solution at some point.